lib/views/screens/loginPage.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+
+class loginPage extends StatelessWidget {
+  const loginPage({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    var screenSize=MediaQuery.of(context).size;
```

Comment:
Let's try to use [Sizer](https://pub.dev/packages/sizer) for responsiveness so we don't have to use Media Query on every page.